config/all.py

```diff
@@ -54,7 +54,6 @@
 # directories to ignore when looking for source files.
 exclude_patterns = [
     'themes',
-    '404.rst'
```

Comment:
No worries, it was a reasonable choice at the time. I had to scour the sphinx docs to find `:orphan: True`